subprojects/logging/src/main/java/org/gradle/internal/logging/services/DefaultLoggingManager.java

```diff
@@ -115,6 +115,12 @@ public LogLevel getLevel() {
     }
 
     @Override
+    public DefaultLoggingManager setMaxWorkerCount(int maxWorkerCount) {
```

Comment:
I'm unsure of calling this feature _max worker count_ as it make less sense when it come to logging. I was thinking dropping _max_ and simply have _worker count_.